specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/virtualNetwork.json

```diff
@@ -861,6 +865,49 @@
       ],
       "description": "ResourceNavigationLink resource."
     },
+    "ServiceDelegationPropertiesFormat": {
+      "properties": {
+        "serviceName": {
+          "type": "string",
+          "description": "The name of the service to whom the subnet should be delegated (e.g. Microsoft.Sql/servers)"
+        },
+        "actions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Describes the actions permitted to the service upon delegation"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the resource."
+        }
+      },
+      "description": "Properties of a service delegation."
+    },
+    "Delegation": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ServiceDelegationPropertiesFormat",
```

Comment:
Please rename `ServiceDelegationPropertiesFormat` to `ServiceDelegationProperties` for brevity